tests/e2e/functional_test.go

```diff
@@ -216,6 +267,8 @@ var _ = Describe("Functional e2e", Serial, func() {
 			return rollout, nil
 		})
 
+		// TODO: update this controller image when Numaflow v1.3.1 is released
+		//       versions prior to v1.3.0 do not contain the MonoVertex CRD and could lead to issues in future tests
 		verifyNumaflowControllerDeployment(Namespace, func(d appsv1.Deployment) bool {
 			return d.Spec.Template.Spec.Containers[0].Image == "quay.io/numaio/numaflow-rc:v0.0.6"
```

Comment:
So, what's happening now? It looks like we are updating to v0.0.6, so how is the code below working? Was MonoVertex CRD in 0.0.6?

Reply:
The tests following this rely on just the Numaplane Controller to update/delete the CRDs. This way, the Numaflow Controller version doesn't impact us. In the future it could if we do any tests checking the MonoVertex pods after updating the controller version.

Reply:
Ahh yes, since the CRDs aren't actually contained within the Numaflow Controller version, that part doesn't matter. We just don't have the MonoVertex being reconciled.

Reply:
In that case, we should update the comment which says "versions prior to v1.3.0 do not contain the MonoVertex CRD and could lead to issues in future" to "versions prior to v1.3.0 do not reconcile MonoVertex"